packages/account/src/Components/proof-of-identity-container/proof-of-identity-container.scss

```diff
@@ -1,17 +0,0 @@
-.manual-poi {
-    .dc-timeline {
-        &__container {
-            flex: 1;
-            max-width: 63.4rem;
-
-            @at-root .manual-poi--mobile .dc-timeline__container {
-                max-width: none;
-            }
-        }
-        &__title {
-            font-size: var(--text-size-s);
-            line-height: 2.4rem;
-            margin: -4px 0 2.4rem;
-        }
-    }
-}
```

Comment:
why are we removing these styles?

Reply:
As I see they are included also here -  poi/status/unsupported/unsupported.scss
so because of duplication

Reply:
can u check with @akmal if it is okay to remove these styles from here?

Reply:
sure, will ask him

Reply:
Hi @yauheni-kryzhyk-deriv , please restore this file. This file is needed to fix webpack issue which doesn't generate css due to some misconfiguration between packages. If we remove this file, all the styles for manual upload will be gone, no matter if they exist in different place.

Reply:
Hi @akmal-binary, Ok, sure I will restore it. 
But for me it sounds weird. When I tried to change something on localhost in this file I didn’t see any changes. And if we will want to change some styles in the future, we have to remember not to forget about two places for changes the same styles? why we don’t have the same issues in other files?

Reply:
BTW, all these changes were done in this PR https://github.com/binary-com/deriv-app/pull/6349
there were no issues from QA side.
Perhaps we can delete this file?